### PR TITLE
add quick-start guide

### PR DESCRIPTION
**Pull request by cdepillabout**
This PR adds a quick-start guide.

The contents of this guide are taken from <https://discourse.nixos.org/t/purenix-nix-backend-for-purescript/15756/3>.

This is for https://github.com/cdepillabout/open-blog-posts/pull/2#discussion_r776315747.

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# Quick Start
+
+Here's a quick-start guide for playing around with PureNix.
```
Comment (jonascarpay) [inline docs/quick-start.md:3]:
```suggestion
Here's a quick-start guide for seting up a PureNix project.
```

File: docs/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+From here, use `spago` to create your PureNix project:
+
+```console
+$ mkdir my-cool-project
+$ cd my-cool-project/
+$ spago init
+```
+
+This creates a few files.  You'll need to edit some of these files in order to
```
Comment (jonascarpay) [inline docs/quick-start.md:37]:
```suggestion
This creates a template project.  You'll need to edit some of these files in order to
```

**Review by jonascarpay — CHANGES_REQUESTED** [review]
I like to format markdown files with one sentence per line, since it gives nice diffs and easy sentence reordering. Do you prefer this way?

**Comment by cdepillabout** [comment]
> I think it makes sense to add a link to the README in this PR.

Good idea, I did this in b6ccae3.

> More generally, I like to format markdown files with one sentence per line, since it gives nice diffs and easy sentence reordering. Do you prefer this way?

I've generally just been keeping to 80-characters per line, but I don't really have a strong feeling about this.

Nice diffs are of course nice though, so I'll switch to one sentence per line.  I made this change in 71a3de4.

File: docs/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+In general, I recommend you take a look at some of our libraries to see exactly
+how they are packaged, and copy what they do.  For instance,
+[`purescript-foldable-traversable`](https://github.com/purenix-org/purescript-foldable-traversable)
+or [`purescript-maybe`](https://github.com/purenix-org/purescript-maybe).
+
+Currently, we only have a
+[temporary package set](https://github.com/purenix-org/temp-package-set).  This
+post will explain how to get started with using it.  There is an
+[open issue](https://github.com/purenix-org/purenix/issues/36) about creating an
+actual package set.
```
[the remark this answers is not in the document]
Comment (cdepillabout) [reply docs/quick-start.md]: Thanks for pointing this out.

I added another section that tries to clarify how package sets work, and what an end-user needs to be aware of with the PureNix package set: 2e952c3

**Comment by cdepillabout** [comment]
I'll go ahead and merge this in, but @jonascarpay if there's anything else you'd like me to change, let me know and I can send a follow-up PR.

**Comment by jonascarpay** [comment]
@cdepillabout Oh, were you waiting for a re-review? Sorry, this looks good, thanks!

**Pull request merged**